tap/extensions/http/handlers.go

```diff
@@ -34,12 +35,13 @@ func handleHTTP2Stream(http2Assembler *Http2Assembler, tcpID *api.TcpID, superTi
 	switch messageHTTP1 := messageHTTP1.(type) {
```

Comment:
messageHTTP1?

Reply:
It's named `messageHTTP1` because [`http2_assembler.go` returns HTTP 1.x data structure](https://github.com/up9inc/mizu/blob/4007b842573832ebe5f3ba1fe988f7302900fe54/tap/extensions/http/http2_assembler.go#L160) from `net/http` package. (`http.Request` and `http.Response`) It returns HTTP 1.x data structure because everything related to HTTP designed around that data structure in HTTP extension or other parts of the software related to HAR conversion etc.